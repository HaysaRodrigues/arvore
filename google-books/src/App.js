import './App.css';
import BooksList from "./books-list/BooksList";
import Search from "./search/Search";

function App() {
  return (
    <div className="App">
      <Search />
      <BooksList />
    </div>
  );
}

export default App;
