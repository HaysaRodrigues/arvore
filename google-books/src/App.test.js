import renderer from 'react-test-renderer';
import App from './App';

test('should renders app', () => {
  const tree = renderer.create(<App />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});