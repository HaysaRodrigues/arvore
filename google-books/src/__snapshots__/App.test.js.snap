// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders app 1`] = `
<div
  className="App"
>
  <div>
    search
  </div>
</div>
`;
